tests/test_base.py

```diff
@@ -70,3 +70,15 @@ def test_finalize_interface(self):
         """
         visualizer = Visualizer()
         self.assertIs(visualizer.finalize(), visualizer.ax)
+
+    def test_size_property(self):
```

Comment:
This tests the setter, but it would be nice to also test the default getter from the current figure. For this test I propose:

1. Creating a new figure using `plt.subplots`
2. Setting the size of the figure 
3. Create a new visualizer 
4. Check that the size of the visualizer is the same as the size of the figure.

Reply:
Looks great, thank you!